contracts/seed/SeedFactory.sol

```diff
@@ -65,10 +65,10 @@ contract SeedFactory is CloneFactory, Ownable {
         address _beneficiary,
         address _admin,
         address[] memory _tokens,
+        address[] memory _whitelists,
         uint256[] memory _softHardThresholds,
         uint256 _price,
-        uint256 _startTime,
-        uint256 _endTime,
+        uint256[] memory _startTimeEndTime,
```

Comment:
Is this done because we reached the max amount of arguments?

Reply:
yes